docusaurus/docs/adding-typescript.md

```diff
@@ -31,7 +31,9 @@ yarn add typescript @types/node @types/react @types/react-dom @types/jest
 
```

Comment:
Can you add this new variable to `advanced-configuration.md`?

Reply:
Hey @ianschmitz , just added this, sorry I forgot to do it the first time around.